### PR TITLE
fix: validate when widget option properties are updated

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Validation was not working when widget options were being added or removed. So I fixed it!

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 1]
```diff
@@ -178,7 +180,23 @@ export default defineComponent<Props>({
...
+                const deletedProperties = xor(oldSchemaProperties, newSchemaProperties);
+                deletedProperties.forEach((propertyName) => {
+                    delete state.schemaFormData[propertyName];
+                });
+            } else if (oldSchemaProperties.length < newSchemaProperties.length) { // add case
+                const addedProperties = xor(oldSchemaProperties, newSchemaProperties);
+                addedProperties.forEach((propertyName) => {
+                    state.schemaFormData[propertyName] = undefined;
+                });
+            }
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:194]: 1. check added / deleted property 
2. update `schemaFormData` (This triggers validation within the `JsonSchemaForm` component.)

Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:194]: 1. check added / deleted property

File: apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 2]
```diff
@@ -178,7 +180,23 @@ export default defineComponent<Props>({
...
+            state.schemaFormData = { ...state.schemaFormData };
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:195]: 2. update `schemaFormData` (This triggers validation within the `JsonSchemaForm` component.)

File: apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:194]: praise: cool

**Review by WANZARGEN — APPROVED** [review]
Cool!

**Review by skdud4659 — APPROVED** [review]
LGTM!!

**Pull request merged**